pkg/asset/installconfig/azure/azure.go

```diff
@@ -0,0 +1,153 @@
+package azure
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/openshift/installer/pkg/types/azure"
+	"github.com/openshift/installer/pkg/types/azure/validation"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	survey "gopkg.in/AlecAivazis/survey.v1"
+)
+
+const (
+	azurePublicCloudTenantID string = "72f988bf-86f1-41af-91ab-2d7cd011db47"
```

Comment:
so this one is actually our tenant ID :P probably need to change this, sorry about that.

Reply:
fixed, added tenantID to the survey questions